src/components/atoms/modal/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames/bind'
+import ReactModal from 'react-modal'
+
+import styles from './styles.css'
+
+class Modal extends React.Component {
```

Comment:
You can import `Component` from react lib and then just you here

`class Modal extends Component`